lib/proto/payment.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+
+option go_package = "./payment";
+
+package payment;
+
+service Payment {
+  rpc Pay(PayRequest) returns (PayResponse) {}
+  rpc AuthorizeCard(AuthorizeCardRequest) returns (AuthorizeCardResponse) {}
+  rpc RegisterRecipient(RegisterRecipientRequest) returns (RegisterRecipientResponse) {}
+  rpc GetTransactionHistory(GetTransactionHistoryRequest) returns (GetTransactionHistoryResponse) {}
+}
+
+message PayRequest {
+}
+
+message PayResponse {
+}
+
+message AuthorizeCardRequest {
+}
+
+message AuthorizeCardResponse {
+}
+
+message RegisterRecipientRequest {
+}
+
+message RegisterRecipientResponse {
+}
+
+message GetTransactionHistoryRequest {
+}
+
+message GetTransactionHistoryResponse {
+}
```

Comment:
add newline at EOF.
btw doesn't Go formatter doesn't automatically do this tho?

Reply:
wait nvm, this is proto file
should still add newline tho